docs/IE_DG/ONNX_Supported_Ops.md

```diff
@@ -1,3 +1,19 @@
+# ONNX format support in the OpenVINO™ {#openvino_docs_IE_DG_ONNX_Supported_Ops}
+
+Beginning from 2020.4 release OpenVINO™ supports to read native ONNX models to nGraph representation.
```

Comment:
Starting from the 2020.4 release, OpenVINO™ supports reading native ONNX models to nGraph representation.

Reply:
I think it should either be "supports reading native ONNX models" or "supports conversion of native ONNX models to nGraph representation".

Reply:
Fixed